python/mlc_chat/rest.py

```diff
@@ -85,65 +83,74 @@ def _parse_args():
 """
 List the currently supported models and provides basic information about each of them.
 """
+
+
 @app.get("/models")
 async def read_models():
-    return {
-        "data": [{
-            "id": model, 
-            "object":"model"
-        } for model in supported_models()]
-    }
+    return {"data": [{"id": model, "object": "model"} for model in supported_models()]}
+
 
 """
 Retrieve a model instance with basic information about the model.
 """
```

Comment:
Comment should be part of read_model function? or is it a convention of fast API, in that case we should try to disable black auto formatter cc @sudeepag